### PR TITLE
feat: storybook config

### PR DESCRIPTION
**Pull request by nconnect1789**
Add storybook to project.
To ran storybook use : CLI yarn storybook.
**Additional info**: regardless of installation storybook, a lot of files appeared automatically in the packages/backend/public/uploads/ folder. I added this folder to the `.gitignore` file. Please check if it's correct.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,16 +5,29 @@
...
   "devDependencies": {
+    "@babel/core": "^7.19.1",
+    "@storybook/addon-actions": "^6.5.12",
+    "@storybook/addon-essentials": "^6.5.12",
+    "@storybook/addon-interactions": "^6.5.12",
+    "@storybook/addon-links": "^6.5.12",
+    "@storybook/builder-webpack5": "^6.5.12",
+    "@storybook/manager-webpack5": "^6.5.12",
+    "@storybook/react": "^6.5.12",
+    "@storybook/testing-library": "^0.0.13",
```
Comment (im-klau) [inline package.json:26]: `design-system` directory should contain all the storybook packages, which means there should be an additional `package.json` file inside it. 

This is just a global one, it doesn't even need to know about any nested packages installed in the project :D

Comment (nconnect1789) [reply package.json]: Updated. Please check it.

File: package.json [hunk 2]
```diff
@@ -13,8 +13,11 @@
     "packages/design-system"
   ],
   "devDependencies": {
+    "@babel/core": "^7.19.1",
+    "babel-loader": "^8.2.5",
```
Comment (im-klau) [inline package.json:17]: you can just remove these two packages, they're in the `design-system` directory already, we don't need them here

Comment (nconnect1789) [reply package.json]: Updated.

Comment (im-klau) [reply package.json]: thank you!

**Review by im-klau — APPROVED** [review]

**Pull request merged**